phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServices.java

```diff
@@ -221,4 +221,8 @@ default void closeAllConnections(SQLExceptionInfo.Builder reasonBuilder) {
     }
 
     PMetaData getMetaDataCache();
+
+    int getConnectionCount();
```

Comment:
just thinking can it be like "int getConnection(bool isInternal);"?

Reply:
Thanks for suggestion, I have updated it.